Composer/packages/client/src/pages/publish/index.tsx

```diff
@@ -339,8 +339,8 @@ const Publish: React.FC<PublishPageProps> = (props) => {
 
       if (result) {
         if (settings.publishTargets && settings.publishTargets.length > index) {
-          const target = settings.publishTargets.slice(0, index).concat(settings.publishTargets.slice(index + 1));
-          await actions.setSettings(projectId, { ...settings, publishTargets: target });
+          const targets = settings.publishTargets.slice(0, index).concat(settings.publishTargets.slice(index + 1));
+          await actions.setPublishTargets(targets);
```

Comment:
Hi @beyackle I found the action use setPublishTarget, use setPublishTarget[s] will throw error. And the arguments is ( _, publishTarget).

Reply:
I fixed these in #3550

Reply:
Thanks for taking care of that.